packages/web/tt.js

```diff
@@ -0,0 +1,543 @@
+  'profile',
+  'settings',
+]
+
+const HEADER_SPEC = [
+  { id: 'namespace', title: 'namespace' },
+  { id: 'fullId', title: 'key' },
+  { id: 'status', title: 'status' },
+  { id: 'source', title: 'Source String' },
+  { id: 'target', title: 'Translated String' },
+  { id: 'tlnotes', title: 'Translator Notes' },
+  { id: 'lastCommitSHA', title: 'Last Commit to Source' },
+]
+
+const computeLocaleCompleteness = (localeReport) => {
+  let total = 0
+  let missing = 0
+  let ood = 0
+
+  for (ns in localeReport.namespaces) {
+    for (translation of localeReport.namespaces[ns]) {
+      total++
+
+      if (translation.status === 'MISSING_TRANSLATION') {
+        missing++
+      } else if (translation.status === 'SOURCE_NEWER') {
+        ood++
+      }
+    }
+  }
+
+  return [
+    `${(100 * missing / total).toFixed(2)}%`,
+    `${(100 * ood / total).toFixed(2)}%`,
+  ]
+}
+
+const generateIndexHTML = async (report) => {
+  const html = `
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Journaly Translations</title>
+    <style type="text/css">
+      body {
+        margin:40px auto;
+        max-width:650px;
+        line-height:1.6;
+        font-size:18px;
+        color:#444;
+        padding:0 10px
+      }
+      h1,h2,h3 {
+        line-height:1.2
+      }
+      table {
+        width: 100%;
+      }
+      thead th {
+        background-color: lightgray;
+      }
+    </style>
+  </head>
+  <body>
+    <header>
+      <h1>Help Translate Journaly!</h1>
+    </header>
+    <p>
+      Journaly is currently accepting translations in the ${LOCALES.length}
+      languages other than English. We'd love to have your help improving these
+      or translating the UI into new languages!
+    </p>
+    <p>
+      To help translate one of the existing languages, find it in the table
+      below, download the linked template, fill it out and email it to
+      <a href="mailto:hello@journaly.com">hello@journally.com</a>.
+    </p>
+    <p>
+      To translate a language that isn't listed, drop us a line at
+      <a href="mailto:hello@journaly.com">hello@journally.com</a>.
+      and we'll look at adding it. Since not every language has the same usage
+      among our userbase, and because each translation requires maintainance
+      effort, we may hold on adding some languages until we find there is a
+      significant demand, but generally we're open to translations. Contact
+      us if you're not sure!
+    </p>
+
+    <h2>Filling out a Translation Template</h2>
+    <p>
+      To fill out a template, simply look at the "Source String" column which
+      contains the original English text, and enter the translation in the
+      "Translated String" column. Some strings contain variables which are
+      represented as "{{variable_name}}", which will be substituted with a
+      value on the site. For example the read time string is
```

Comment:
Something to think about is that this particular doc is likely to be read by a lot of non-developers and also non-native English speakers, so I think the term "string" could be confusing for some people. It is in an important column name, though, so I'm wondering it maybe we should just add a little note to clarify for people just in case?